mantidproject/mantid
# Replace the splash screen with a celebration one

Replaced the splash screen with a celebration one.
Also add the layered image file (Paint dot net) to allow easier editing in the future

Issue #19897 has been created to remove the balloons for the subsequent release

closes #19896

**To test:**


1. Startup Mantidplot, look at the splash screen
1. See Balloons, feel happy
1. Check that the version label at the bottom of the splash screen does not run over the image.


**Release Notes** 

 docs/source/release/index.rst


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
